e2e/workspace-create/src/create-nx-plugin.test.ts

```diff
@@ -26,10 +26,11 @@ describe('create-nx-plugin', () => {
     checkFilesExist(
       'package.json',
       packageManagerLockFile[packageManager],
-      `packages/${pluginName}/package.json`,
-      `packages/${pluginName}/project.json`
+      `project.json`,
+      `generators.json`,
+      `executors.json`
     );
 
-    expect(() => runCLI(`e2e ${pluginName}-e2e`)).not.toThrow();
+    expect(() => runCLI(`e2e e2e`)).not.toThrow();
```

Comment:
Is this a typo or does the e2e project gets generated with the `e2e` name?

Reply:
This is on purpose - the project is named e2e as is the target. It mirrors what we do for react standalone